repository/SmalltalkCI-Core.package/SCIMetacelloLoadSpec.class/instance/ensureAuthor.st

```diff
@@ -0,0 +1,4 @@
+initialize-release
+ensureAuthor
+	SmalltalkCI getAuthor ifEmpty: [
+		self class setAuthor: self class defaultAuthor ]
```

Comment:
I have removed the preserving because it's not needed anymore. `SCIMetacelloLoadSpec` now only needs to make sure that the author is not empty, because that can cause loading or testing issues. Changing the build process as described in https://github.com/hpi-swa/smalltalkCI/pull/162#issue-159992343 makes all the `finishUpAfterCI` logic redundant.